src/style.css

```diff
@@ -0,0 +1,25 @@
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.starts__star {
+  display: flex;
+  width: 16px;
+  height: 16px;
+  background-image: url("../src/images/star.svg");
+  background-position: center;
+  background-repeat: no-repeat;
+  margin-right: 4px;
+}
+
+.starts__star:last-child {
+  margin-right: 0;
+}
+
+.is-active {
+  background-image: url("../src/images/star-active.svg");
+}
```

Comment:
You can add the active star by modifier (`stars--1`)
And remember about checklist p2
![image](https://github.com/mate-academy/layout_stars/assets/84296041/d9ae8277-4ded-4d33-aaab-98abad37149c)


```suggestion
.stars.stars--1 :nth-child(-n+1) {
  background-image: url("../src/images/star-active.svg");
}
```